Rakefile

```diff
@@ -32,6 +32,7 @@ if (Rails.env.development? || Rails.env.test?) && defined? Dummy
 
   task(:default).clear
   if ENV['APPRAISAL_INITIALIZED'] || ENV['CI']
+    tasks = [:spec]
```

Comment:
All this time.......we weren't actually running tests in CI (!) This would fail silently afaict. I stumbled upon this by trying to run `be rake` locally.

Reply:
😱